frontend/src/screens/GameScreen/CustomButton.tsx

```diff
@@ -0,0 +1,23 @@
+import React, { ReactNode } from "react";
+import { TouchableOpacity, Text, StyleSheet, View, StyleSheetProperties, StyleProp, ViewStyle } from "react-native";
+
```

Comment:
I think it may be better if CustomButton, PrimaryButton, and Secondary Button were in a new directory `src/components/Button` rather than in the `GameScreen` directory, since these buttons might be used elsewhere in the app. The existing TextIconButton could be in that directory as well.